package.json

```diff
@@ -73,6 +73,7 @@
     "@lerna/publish": "^5.1.8",
     "@lerna/run": "^5.1.8",
     "@nrwl/nx-cloud": "latest",
+    "@types/debug": "^4.1.7",
```

Comment:
this should be in the devDependencies of the package that imports debug so `utils`